### ISSUE TITLE
add tournesol.app

### ISSUE THREAD

**Issue by rambip** [sampart/awesome-tech-ethics-resources#7]
Tournesol (https://tournesol.app/) is the most promising project to change the youtube recommandation algorithm.

The community is led by researchers and is very active.

**Comment by sampart** [comment]
Thanks for this.

Here's an extract from their FAQ, so it's clear what I'm primarily referencing in my comments (although I have looked around most of the site, not just this):

> **What is the goal of the Tournesol project?**
> Tournesol is a participatory research project led by the Tournesol Association. Recognizing the importance of sharing common interest information on a large scale, its goal is to advance research in the ethics of algorithms and recommendation systems.
> 
> To achieve this goal, we are developing the Tournesol platform where all contributors are invited to build together a large open database of video quality judgments.
> 
> The immediate effect of this database is to improve the recommendations of thousands of people using the platform. The long-term effect is to facilitate and improve future research projects by making this database available.

It seems like they have some interesting ideas, but I've decided not to add this project to the list. The [focus of this list](https://github.com/sampart/awesome-tech-ethics-resources/blob/main/CONTRIBUTING.md) is primarily on things that can help those involved in technological development achieve this mitigation/avoidance of the negative effects of technology on society. The initial work of Tournesol ("a large open database of video quality judgments") is too specific and user-facing (as opposed to creator-facing) to relate to this goal. The longer-term work ("facilitate and improve future research projects by making this database available") is more relevant, but in reality this list would be better served by linking to action-oriented research which benefits from Tournesol rather than Tournesol itself.

Thanks for getting involved, though. And I've added [your other suggestion](https://github.com/sampart/awesome-tech-ethics-resources/issues/8) as I guess you've seen.
